bin/osg-topology

```diff
@@ -178,14 +172,16 @@ def main():
         with open(args.email_file, 'r') as f:
             contents = f.read()
             args.contact_emails = contents.split('\n')
-    except TypeError:
+    except (AttributeError, TypeError):
         pass  # --email-from-file not specified
     except IOError as exc:
         sys.exit("Failed to read email file: %s" % exc)
+    except UnicodeDecodeError as exc:
+        sys.exit("Invalid unicode in email file: %s" % exc)
 
-    commands = {"list_resource_contacts": list_resource_contacts,
-                "list_vo_contacts": list_vo_contacts,
-                "list_contacts": list_contacts,
+    commands = {"list-resource-contacts": list_resource_contacts,
+                "list-vo-contacts": list_vo_contacts,
+                "list-contacts": list_contacts,
```

Comment:
does this change (underscores to dashes) break anything?  Or is it just the user CLI?

Reply:
It's necessary because of the
```
subparsers = oparser.add_subparsers(dest="which")
```
line. This causes argparse to put the name of the subparser into `args.which`. The previous code manually added a default value for `args.which` using the underscore version. It's not user-visible.